airflow/models/dagbag.py

```diff
@@ -446,14 +446,6 @@ def collect_dags(
                 self.log.exception(e)
         self.dagbag_stats = sorted(
             stats, key=lambda x: x.duration, reverse=True)
-        for file_stat in self.dagbag_stats:
```

Comment:
I am guessing we don't use this metric at all  currently at Lyft ?

Reply:
Yeah I don't think we use that metrics

Reply:
cool, thanks :)